.spelling

```diff
@@ -34,6 +36,7 @@ CEP
 changesets
 chromehangs
 CircleCI
+clientid
```

Comment:
Let's mark this as a technical term instead of whitelisting it in the spell checker to be consistent with the existing docs. I.e. remove this line and use ``` `client_id` ``` in the articles.

Reply:
sounds good